src/workspacer.Native/Native/Win32/Win32.Long.cs

```diff
@@ -10,7 +10,7 @@ namespace workspacer
     public static partial class Win32
     {
         [Flags]
-        public enum WS : ulong
+        public enum WS : long
```

Comment:
Any reason this needed to be change from `ulong` to regular `long`?

Reply:
Browsing through docs.microsoft.com, the signatures of functions like [`SetWindowLongPtr`](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setwindowlongptra) take a `LONG_PTR`. 

[`LONG_PTR`](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/bbcfb0af-8349-4e98-ad26-957e1363f714) is (logically) a pointer for the long type. 

[`LONG`](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/29d44d70-382f-4998-9d76-8a1fe93e445c) is a 32-bit **signed** integer.

In .NET, [`long`](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/integral-numeric-types) is a signed 64-bit integer. I changed it to be a _bit_ more correct (in my opinion), by allowing negative values (which are pretty unlikely, though I am no expert in this sort of Windows programming).

However, if you think it'd better to keep it as `ulong`, I can change it back.

Reply:
Given the currently used values I don’t think we gain much either way. But no biggie 🙂